packages/pluggableWidgets/rich-text-web/src/components/__tests__/RichText.spec.tsx

```diff
@@ -0,0 +1,156 @@
+import { createElement } from "react";
```

Comment:
You removed the DOM structure test, but we need it for backwards compatibility reasons. Can you add it back and fix it?

Reply:
The is no backward compatibility, we changed the library, it renders differently.
Please correct me if I am wrong.

Reply:
I don’t think it’s possible to render ckeditor structure in snapshot.
Even for the future you will not have access to ckeditor dom structure, even with that if they change version with structure your test with fail without your control.

The only snapshot which is possible here is outside div with it's classname, and I don't think that is a blocker

Reply:
Added minimal snapshot testing